cli/oci.go

```diff
@@ -339,3 +343,63 @@ func getCgroupsDirPath(mountInfoFile string) (string, error) {
 
 	return cgroupRootPath, nil
 }
+
+// This function assumes it should find only one file inside the container
+// ID directory. If there are several files, we could not determine which
+// file name corresponds to the sandbox ID associated, and this would throw
+// an error.
+func fetchContainerIDMapping(containerID string) (string, error) {
+	if containerID == "" {
+		return "", fmt.Errorf("Missing container ID")
+	}
+
+	dirPath := filepath.Join(ctrsMapTreePath, containerID)
+	if _, err := os.Stat(dirPath); os.IsNotExist(err) {
```

Comment:
You can get rid of this check. The ReadDir call below will return an error if the directory does not exist.

Reply:
Yes, indeed.